.gitlab-ci.yml

```diff
@@ -7,7 +7,7 @@ stages:
     - deploy
 
 variables:
-    EXCLUDE_PYTHON: 37
+    EXCLUDE_PYTHON: 37, 38, 39, 310, 311
```

Comment:
Where is this variable used?

Reply:
In this case, the unittest in Gitlab will only test for 312 and coverage, so that the pipline will run faster